### PR TITLE
Fixed typos, grammar, spelling, and formatting.

### PR DESCRIPTION
**Pull request by alyashour**
I have fixed grammatical errors, sentence structure, etc.
To best suit the project, I have not changed any of the meaning and have strictly kept with what was already there.
I'm not sure if this is the only file this information is in. If there is a better way to implement these changes let me know, I'd love to do that.

### CODE REVIEW

**Comment by collin-marando** [comment]
This is really great work!

File: kubejs/assets/twr_quests/lang/en_us.json [hunk 1]
```diff
@@ -14,7 +14,7 @@
   "chapter.twr.1E695E6D1F5B227A.title": "A Day of Craftsman",
   "chapter.twr.3A3B51CF7F606665.subtitle": "This is not the steam age you remembered",
   "chapter.twr.3A3B51CF7F606665.title": "Steam Powered",
-  "chapter.twr.45227BBF4538D5BC.subtitle": "You landed from the space station to the frozen Earth, please be prepared, this is a difficult mission!",
+  "chapter.twr.45227BBF4538D5BC.subtitle": "You landed onto a frozen Earth, please be prepared, this is a difficult mission!",
```
Comment (yuesha-yc) [inline kubejs/assets/twr_quests/lang/en_us.json:17]: The space station is part for lore so I suggest not removing this.

File: kubejs/assets/twr_quests/lang/en_us.json [hunk 2]
```diff
@@ -218,35 +218,35 @@
...
+  "quest.twr.19038BB5A9E42055.description.4": "&6&lIf everything is correct and the fuel is inserted, an indicator in the bottom left of UI will light up.",
   "quest.twr.19038BB5A9E42055.subtitle": "Even hotter topics",
   "quest.twr.19038BB5A9E42055.title": "Steelmaking crucible",
-  "quest.twr.190859211258D56B.description.1": "Bronze axe can chop down an entire tree at once! Hold sneak to disable this function. Highly recommended.",
+  "quest.twr.190859211258D56B.description.1": "A Bronze axe can chop down an entire tree at once! If you ever feel like wasting time instead, hold sneak to chop normally.",
   "quest.twr.190859211258D56B.subtitle": "Also an upgrade tool",
   "quest.twr.1938FC4AAD99137E.0A0A4C059B9D1326.title": "Effect",
-  "quest.twr.1938FC4AAD99137E.description.1": "Most of research puzzle will have a certain answer to the completion of the research paper.",
+  "quest.twr.1938FC4AAD99137E.description.1": "Most of the research puzzle will have a certain answer to the completion of the research paper.",
```
Comment (yuesha-yc) [inline kubejs/assets/twr_quests/lang/en_us.json:228]: research puzzles

File: kubejs/assets/twr_quests/lang/en_us.json [hunk 3]
```diff
@@ -520,25 +520,25 @@
...
-  "quest.twr.3B5663CF194B054C.description.2": "When your body temperature get to a dangerous situation, you experience bar would become &dbody temperature indicator&r, &c&land you would get hurt after it become completly red&r.",
-  "quest.twr.3B5663CF194B054C.description.3": "You should find a &6&lShelter from wind and rain&r ASAP and &6&llight up a campfire&r to maintain body temperature.",
-  "quest.twr.3B5663CF194B054C.description.4": "Avoid drenched in snow or soaked in water, this make you become &b&lwet&r and easily lost body temperature.",
+  "quest.twr.3B5663CF194B054C.description.1": "In this icy world, It's important to watch your body temperature.",
+  "quest.twr.3B5663CF194B054C.description.2": "When your body temperature gets to a dangerous level, you experience bar will become a &dbody temperature indicator&r, &c&land you'll start taking damage after it turns completly red&r.",
+  "quest.twr.3B5663CF194B054C.description.3": "You should find a &6&lShelter from wind and rain&r ASAP and &6&llight up a campfire&r to stay warm.",
+  "quest.twr.3B5663CF194B054C.description.4": "Avoid getting drenched in snow or soaked in water, this will get you &b&lwet&r.",
```
Comment (yuesha-yc) [inline kubejs/assets/twr_quests/lang/en_us.json:531]: I suggest leaving the "easily lose body temperature"

File: kubejs/assets/twr_quests/lang/en_us.json [hunk 4]
```diff
@@ -799,17 +799,17 @@
   "quest.twr.5F455ED38549D9EF.subtitle": "Not released yet",
   "quest.twr.5F455ED38549D9EF.title": "Tier Three Generator",
   "quest.twr.6000F43C135FC068.description.1": "The power generated by the hand crank is not much, but with a large cogwheel, it is enough to pump steam out and run a bronze steam engine.",
-  "quest.twr.6000F43C135FC068.description.2": "After setting up the steam engine, you can free your hands.",
+  "quest.twr.6000F43C135FC068.description.2": "After setting up the steam engine it will run itself.",
```
Comment (yuesha-yc) [inline kubejs/assets/twr_quests/lang/en_us.json:802]: I suggest using the original phrase

**Comment by yuesha-yc** [comment]
@alyashour I've requested some changes, thanks for your hard work!

**Pull request merged**